database/src/lib.rs

```diff
@@ -2,6 +2,7 @@ use bitflags::bitflags;
 
 mod error;
 pub mod mdbx;
+pub(crate) mod metrics;
```

Comment:
```suggestion
mod metrics;
```
Is there a reason for all these `pub(crate)` annotations? E.g. here, it could simply be dropped.